source/Nevermore/Diagnostics/DefaultTransactionLogger.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Nevermore.Diagnositcs;
+
+namespace Nevermore.Diagnostics
+{
+    public class DefaultTransactionLogger : ITransactionLogger
+    {
+        static readonly ILog Log = LogProvider.For<DefaultTransactionLogger>();
+
+        readonly long infoThreshold;
+
+        public DefaultTransactionLogger(long infoThreshold = 10_000)
+        {
+            this.infoThreshold = infoThreshold;
+        }
+        
+        public void Write(long duration, string transactionName)
+        {
+            var level = duration >= infoThreshold ? LogLevel.Info : LogLevel.Debug;
+            Log.Log(level, () => $"Transaction '{transactionName}' was open for {duration}ms");
```

Comment:
Yeah saw this in `DefaultQueryLogger` which I followed but thought it was weird. I've now made `DefaultTransactionLogger` do nothing but kept the class so that `RelationalStoreConfiguration` constructor has a placeholder to use.

Reply:
Yeh empty class is 👍 . The DefaultQueryLogger replicated the the behaviour that was there before it was possible to override it.